package.json

```diff
@@ -1,6 +1,8 @@
 {
   "devDependencies": {
-    "@salesforce/sfdx-lwc-jest": "^0.7.1"
+    "@sa11y/jest": "^0.1.6-beta.0",
```

Comment:
@davidmreed , @bdvorachek : the repo has sa11y dependency added and it requires  "jest": "^26.0.1". Please can you review changes in the config and the "yarn.lock" to ensure package is correctly updated?
CC @psadie-sfdo , @kenlewis-sfdo , @vmartinov-sforg

Reply:
I don't see any immediate problems, and the Jest action is still passing.

Reply:
Thank you @davidmreed !

Reply:
@diego-caballero , @force2b : I had to revert back versions since the lwc-jest 0.10.2 does not work when the LWC has wire function (to get the record or the SObject info). Most importantly, I changed back version in `sfdx-project.json` from 50 to 48. What was the reason to use the specific version? 
CC @bdvorachek , @andrewyu-salesforce , @npsp-reedestockton